Makefile

```diff
@@ -280,29 +278,6 @@ errcheck:
 	@echo "Running errcheck..."
 	@errcheck $(GO_PACKAGES)
 
-.PHONY: revive
-revive:
```

Comment:
I could see why we would remove revive, I'm not sure if it actually could be a drop-in as I know from golangci-lint that they don't always have 100% configuration as if you have with cmd command.

Reply:
maybe we should integrate revive in golangci-lint?